_partials/_install-self-hosted-docker-based.mdx

```diff
@@ -0,0 +1,34 @@
+<Procedure>
+
+1.  At the command prompt, run the TimescaleDB Docker image:
+
+    ```bash
+    docker pull timescale/timescaledb-ha:pg16
+    ```
+
+1. Run the container:
+
+    ```bash
+    docker run -d --name timescaledb -p 5432:5432 -e POSTGRES_PASSWORD=password timescale/timescaledb-ha:pg16
+    ```
+
+1.  **Tune your PostgreSQL instance for TimescaleDB**
+
+    1. Open a bash session inside the container:
+
+       ```bash
+       docker exec -it timescaledb /bin/bash
+       ```
+
+    1. Tune your database:
+
+       ```bash
+       sudo timescaledb-tune
+       ```
+
+    This script is included with the `timescaledb-tools` package when you install TimescaleDB.
+    For more information, see [configuration][config].
+
```

Comment:
I am 99% sure timescaledb-tune is run automatically on startup in  `timescale/timescaledb-ha:pg16` so this whole section can be deleted

Reply:
Thank you. Removed.